brokerapi/brokers/gcp_service_broker.go

```diff
@@ -485,16 +485,20 @@ func GetCredentialsFromEnv() (models.GCPCredentials, error) {
 	return g, nil
 }
 
-func getStaticPlans() (map[string][]models.ServicePlan, error) {
+func getPlansFromEnv(envVar string) (map[string][]models.ServicePlan, error) {
```

Comment:
Now that you've got an awesome consistent format, storing them all in the same way, you should fold all the loading into a single call.

suggest something like this:

```go
func addPlans(planJson string, plans map[string][]models.ServicePlan) error
// ...

plans := map[string][]models.ServicePlan{}
for _, env := range([]string{"PRECONFIGURED_PLANS", "CLOUDSQL_CUSTOM_PLANS", ...}) {
   if err := addPlans(os.GetEnv(env), plans) {
   //.. err stuff   
   }
}
```